dashboards/google-cloud-composer/cloud-composer-monitor.json

```diff
@@ -0,0 +1,389 @@
+{
+  "category": "CUSTOM",
+  "dashboardFilters": [
+    {
+      "filterType": "RESOURCE_LABEL",
+      "labelKey": "project_id",
+      "templateVariable": "Project"
+    },
+    {
+      "filterType": "RESOURCE_LABEL",
+      "labelKey": "environment_name",
+      "templateVariable": "Cluster"
+    }
+  ],
+  "displayName": "sb-data-projects-composer-load-test-v1",
```

Comment:
This should have a different display name, something like "Cloud Composer Monitor"?

Reply:
Specifically, it should match the displayName field in metadata.yaml

Reply:
done.